### PR TITLE
[Landing page redesign] Mobile nav menu + Downloads Page

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
Screen Shot 2022-06-23 at 11 28 44 AM

screencapture-localhost-3001-download-2022-06-23-11_26_43

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

File: packages/web/src/components/public-site/NavBanner.module.css [hunk 1]
```diff
@@ -128,8 +125,7 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  color: #480b79;
-  font-family: Gilroy !important;
+  color: #7e1bcc;
```
Comment (Kyle-Shanks) [inline packages/web/src/components/public-site/NavBanner.module.css:128]: should be secondary color var?

File: packages/web/src/components/public-site/NavOverlay.module.css [hunk 2]
```diff
@@ -57,51 +58,51 @@
...
+  padding: 13px 0px;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 22px;
   text-align: center;
+  border-radius: 6px;
+  background-color: #ffffff;
+  box-shadow: 0px 0px 1px -2px rgba(133, 129, 153, 0.1),
+    0px 1px 0px -2px #e3e3e3, 1px 2px 5px rgba(133, 129, 153, 0.25);
+  color: #7e1bcc;
```
Comment (Kyle-Shanks) [inline packages/web/src/components/public-site/NavOverlay.module.css:82]: should use color vars here too

File: packages/web/src/pages/download-page/DownloadPage.tsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    <ParallaxProvider>
+      <div
+        id='downloadPage'
+        className={cn(styles.container, { [styles.isMobile]: props.isMobile })}
+      >
+        {showCookieBanner && (
+          <CookieBanner
+            isMobile={props.isMobile}
+            isPlaying={false}
+            // @ts-ignore
```
Comment (Kyle-Shanks) [inline packages/web/src/pages/download-page/DownloadPage.tsx:135]: what was happening here?

**Review by Kyle-Shanks — APPROVED** [review]
nice 👍

File: packages/web/src/components/public-site/handleClickRoute.ts [hunk 4]
```diff
@@ -33,8 +37,10 @@ export const handleClickRoute = (
   // Http(s) routes and landing page routes should trigger a full window reload
   // They are external domains, or in the case of landing pages, we want
   // to load in from 0-state (otherwise it will get picked up by the app router)
+  console.log(route)
   if (route.startsWith('http') || LANDING_PAGE_ROUTES.has(route)) {
     pushWindowRoute(route)
+    console.log('here')
```
[the remark this answers is not in the document]
Comment (nicoback2) [reply packages/web/src/components/public-site/handleClickRoute.ts:43]: whoops

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--mobile-menu-website

**Pull request merged**